rskj-core/src/main/java/co/rsk/Start.java

```diff
@@ -173,6 +173,7 @@ public void startNode(String[] args) throws Exception {
 
         messageHandler.start();
         txHandler.start();
+        channelManager.start();
```

Comment:
`channelManager.start()` should go before `messageHandler.start()`. Whether it was executed before or after `txHandler.start()` was undefined, so you can do whatever sounds better.